host-bin/enter-chroot

```diff
@@ -436,9 +445,22 @@ elif [ ! -f "$shares" ]; then
 downloads ~/Downloads
 EOF
 fi
+
+activeuser=$(getactiveuser)
+if [ -n "$firstrun" ]; then
+    # Store the active user for possible use in other scripts, etc.
+    activeuserfile="$(fixabslinks '/etc/crouton/user')"
+    echo $activeuser > $activeuserfile
```

Comment:
I would rather not (easily) expose this information to the chroot.  Is there a real use for it?

Reply:
Just a nice-to-have. Removed...